playground/assets/__tests__/assets.spec.ts

```diff
@@ -86,7 +87,8 @@ describe('raw references from /public', () => {
   })
 })
 
-test('import-expression from simple script', async () => {
+// This feature seems to have a bug: it doesn't rewrite in dev, so it won't work if the base is /foo/bar/
+test.skip('import-expression from simple script', async () => {
```

Comment:
There was two bugs that was found by this PR. I fixed one but didn't fix the other.
This bug can be reproduced by changing the base to `/foo/bar` in the assets playground with 4.4.9.
related: #6525